File: AtmLocator/app/src/main/java/piotrek/atmlocator/orm/BankDao.java
package piotrek.atmlocator.orm;

import com.j256.ormlite.dao.BaseDaoImpl;
import com.j256.ormlite.support.ConnectionSource;
import com.j256.ormlite.table.DatabaseTableConfig;

import java.sql.SQLException;

/**
 * Created by Admin on 2016-04-26.
 */
public class BankDao extends BaseDaoImpl<Bank, Integer> {
    protected BankDao(Class<Bank> dataClass) throws SQLException {
        super(dataClass);
    }

    protected BankDao(ConnectionSource connectionSource, Class<Bank> dataClass) throws SQLException {
        super(connectionSource, dataClass);
    }

    protected BankDao(ConnectionSource connectionSource, DatabaseTableConfig<Bank> tableConfig) throws SQLException {
        super(connectionSource, tableConfig);
    }
}

File: AtmLocator/app/src/main/java/piotrek/atmlocator/orm/BankDao.kt
package piotrek.atmlocator.orm

import com.j256.ormlite.dao.BaseDaoImpl
import com.j256.ormlite.support.ConnectionSource
import com.j256.ormlite.table.DatabaseTableConfig

import java.sql.SQLException

/**
 * Created by Admin on 2016-04-26.
 */
class BankDao : BaseDaoImpl<Bank, Int> {
    @Throws(SQLException::class)
    protected constructor(dataClass: Class<Bank>) : super(dataClass) {
    }

    @Throws(SQLException::class)
    protected constructor(connectionSource: ConnectionSource, dataClass: Class<Bank>) : super(connectionSource, dataClass) {
    }

    @Throws(SQLException::class)
    protected constructor(connectionSource: ConnectionSource, tableConfig: DatabaseTableConfig<Bank>) : super(connectionSource, tableConfig) {
    }
}
